common/reviews/api/heft.api.md

```diff
@@ -220,7 +220,9 @@ export interface IMetricsData {
     machineOs: string;
     machineProcessor: string;
     machineTotalMemoryMB: number;
+    secondsBeforeFirstExecution: number;
```

Comment:
beforeFirstExecutionSeconds?

Reply:
Or more explicit, `bootSeconds`

Reply:
Also should this have a millis version as well?

Reply:
Updated to `bootDurationMs`.